src/main/kotlin/no/nav/familie/ef/iverksett/journalføring/DistribuerVedtaksbrevTask.kt

```diff
@@ -16,13 +18,18 @@ import java.util.*
                      settTilManuellOppfølgning = true,
                      triggerTidVedFeilISekunder = 15 * 60L,
                      beskrivelse = "Distribuerer vedtaksbrev.")
-class DistribuerVedtaksbrevTask(val journalpostClient: JournalpostClient) : AsyncTaskStep {
+class DistribuerVedtaksbrevTask(val journalpostClient: JournalpostClient,
+                                val lagreTilstandService: LagreTilstandService
+) : AsyncTaskStep {
 
     private val logger: Logger = LoggerFactory.getLogger(this::class.java)
 
     override fun doTask(task: Task) {
         val taskData = objectMapper.readValue<DistribuerVedtaksbrevTaskData>(task.payload)
         val bestillingId = journalpostClient.distribuerBrev(taskData.journalpostId)
+        lagreTilstandService.lagreDistribuerVedtaksbrevResultat(behandlingId = taskData.behandlingId,
```

Comment:
Nå som vi har journalpostId i resultatobjektet ønsker jeg at vi bytter ut payloaden `DistribuerVedtaksbrevTaskData` med `behandlingId` og bruker behandlingId tvers igjennom taskene.

Reply:
mao, `lagreTilstandService.hentJournalpostResultat(behandlingId).journalpostId` for å finne journalpostId-en